core/commands/diff.py

```diff
@@ -205,9 +265,26 @@ def _run(self):
         self.view.settings().set("git_savvy.diff_view.raw_diff", diff)
         text = prelude + '\n--\n' + diff
 
+        added_regions = []
+        removed_regions = []
+
+        if word_diff_regex:
+            def extractor(match):
+                a, b = match.span()
+                regions = added_regions if match.group()[1] == '+' else removed_regions
+                offset = len(added_regions) + len(removed_regions)
+                regions.append(sublime.Region(a - offset * 4, b - (offset + 1) * 4))
```

Comment:
Either I'm slow or it's late, or both.. but I can't figure out why we multiply by 4 ... any chance we could add a docstring here?